x/operator/keeper/dogfood.go

```diff
@@ -0,0 +1,425 @@
+package keeper
```

Comment:
Instead of calling this file `dogfood.go`, please rename it to `consensus_keys.go` or similar. These keys will be used by app chains as well as dogfood.

Reply:
Yeah